Automattic/woocommerce-payments
# Reporting: Add a parent wrapper component for Payment Activity

Fixes #8389 

#### Changes proposed in this Pull Request

* The PR creates a parent wrapper component for `Payment Activity` widget, that will be visible on the `Payment Overview` page ( Payments > Overview ) 
* The component will serve as a wrapper for other sub-components forming the Payment Activity widget. Other PR's will add these components subsequently
* The PR also hides the `Payment Activity` widget behind a feature flag - `_wcpay_feature_payment_overview_widget`. It will appear only when this flag is set to true. ( meant to hide the widget during development phase ) 

#### Additional context
* The PR extracts relevant code from the awesome work of @elazzabi in 
#8234 
* Phase 1 Design: p6riRB-9OY-p2
* Simplified Iteration 1 Design: [bIfbBgyBPRzvXEl7dEg3cv-fi-1373_37800](https://href.li/?https://www.figma.com/file/bIfbBgyBPRzvXEl7dEg3cv/?node-id=1373-37800)

#### Testing instructions
* Checkout this branch on your local client test site
* Browse to the Payments overview page - `Payments > Overview` 
* Without the feature flag added / enabled yet, you will see the overview page without the Payment Activity wrapper. Similar to the screenshot below:

<img src="https://github.com/Automattic/woocommerce-payments/assets/4162931/2316b690-59a9-44cf-bc67-ca1e8ac729a9" width="600" /><br>

* Via CLI, add the feature flag to the client site using the following command:
`wp option update _wcpay_feature_payment_overview_widget 1`
* Reload the Payments Overview page and you will now see the empty Payment Activity wrapper with only the title.

<img src="https://github.com/Automattic/woocommerce-payments/assets/4162931/99d9098c-91b9-4c9c-a407-97690515280b" width="600" />


-------------------


**Post merge**
